sources/Inputs/src/NetworkManager.cpp

```diff
@@ -152,21 +152,20 @@ NetworkManager::buildTree() {
 
     const auto& svarcs = networkVL->getStaticVarCompensators();
     for (const auto& svarc : svarcs) {
+      bool defineSvarcModel = true;
       if (!svarc->getInitialConnected()) {
-        continue;
-      }
-      if (!svarc->hasStandbyAutomaton()) {
+        defineSvarcModel = false;
```

Comment:
Add a warning message like the others for compliance in this specific case

Reply:
Message added